x-pack/plugins/apm/public/components/shared/AgentIcon/icons/php.svg

```diff
@@ -1,18 +1,5 @@
-  <defs>
-    <radialGradient id="paint0_radial" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="translate(9.786 10.326) scale(20.4194)">
-      <stop stop-color="#AEB2D5"/>
-      <stop offset=".3" stop-color="#AEB2D5"/>
-      <stop offset=".75" stop-color="#484C89"/>
-      <stop offset="1" stop-color="#484C89"/>
-    </radialGradient>
-  </defs>
+<svg width="32" height="32" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
+<path d="M3.37399 11.1553H7.82314C9.12901 11.1662 10.0753 11.5408 10.662 12.2786C11.2487 13.0164 11.4422 14.0241 11.243 15.3017C11.1656 15.8855 10.994 16.4581 10.7284 17.0198C10.4737 17.5815 10.1198 18.088 9.66589 18.5396C9.11241 19.1122 8.5204 19.4756 7.88955 19.6299C7.2587 19.7842 6.60561 19.8612 5.93059 19.8612H3.93843L3.30758 23H1L3.37399 11.1553V11.1553ZM5.31634 13.0385L4.32026 17.9945C4.38667 18.0054 4.45307 18.011 4.51948 18.011C4.59685 18.011 4.67453 18.011 4.7519 18.011C5.81438 18.0219 6.69989 17.9175 7.40811 17.6971C8.11633 17.4658 8.59244 16.662 8.83582 15.2852C9.03504 14.1288 8.83582 13.4624 8.23817 13.2863C7.65149 13.1102 6.91571 13.0276 6.0302 13.0385C5.89739 13.0494 5.77022 13.0551 5.64837 13.0551C5.5378 13.0551 5.42159 13.0551 5.29974 13.0551L5.31634 13.0385"/>
+<path d="M13.8715 8H16.1625L15.5151 11.1553H17.5736C18.7025 11.1774 19.5436 11.4087 20.097 11.8491C20.6615 12.2895 20.8275 13.1267 20.5951 14.3601L19.4828 19.8612H17.1586L18.2211 14.6079C18.3317 14.0572 18.2985 13.6663 18.1215 13.435C17.9445 13.2037 17.5627 13.0881 16.976 13.0881L15.1332 13.0716L13.7719 19.8612H11.481L13.8715 8Z"/>
+<path d="M23.0551 11.1553H27.5043C28.8102 11.1662 29.7564 11.5408 30.3431 12.2786C30.9298 13.0164 31.1234 14.0241 30.9242 15.3017C30.8468 15.8855 30.6751 16.4581 30.4095 17.0198C30.1549 17.5815 29.8009 18.088 29.347 18.5396C28.7936 19.1122 28.2015 19.4756 27.5707 19.6299C26.9398 19.7842 26.2868 19.8612 25.6117 19.8612H23.6196L22.9887 23H20.6812L23.0551 11.1553V11.1553ZM24.9975 13.0385L24.0014 17.9945C24.0678 18.0054 24.1342 18.011 24.2006 18.011C24.278 18.011 24.3557 18.011 24.433 18.011C25.4955 18.0219 26.381 17.9175 27.0893 17.6971C27.7975 17.4658 28.2736 16.662 28.517 15.2852C28.7162 14.1288 28.517 13.4624 27.9193 13.2863C27.3326 13.1102 26.5969 13.0276 25.7113 13.0385C25.5785 13.0494 25.4514 13.0551 25.3295 13.0551C25.2189 13.0551 25.1027 13.0551 24.9809 13.0551L24.9975 13.0385"/>
```

Comment:
I think this file can be deleted since it looks like the svg is now embedded in the react component (php.tsx)

Reply:
Right, and the same for `x-pack/plugins/apm/public/components/shared/AgentIcon/icons/rumjs.svg` when we convert to a .tsx file